src/itype.h

```diff
@@ -80,10 +80,9 @@ struct islot_armor {
      */
     std::bitset<num_bp> covers;
     /**
-     * Bitfield of enum body_part
-     * TODO: document me.
+     * Whether this item can be worn on either side of the body
      */
-    std::bitset<num_bp> sided;
+    int sided = false;
```

Comment:
What's a 'left-handed' trait going to do other than apply an arbitrary
penalty for using a handful (ha) of items until the player can find or make
the rare left-handed version? Doesn't seem fun or challenging.